### PR TITLE
Improve reliability

### PR DESCRIPTION
**Pull request by RenaultR42**
Handle more errors
Stop the application if any subprocess is died
Add a systemd service, in order to start it on boot or to restart it if the application stopped

### CODE REVIEW

File: ventilator_websocket.py [hunk 1]
```diff
@@ -47,14 +47,18 @@ def run(self, name):
...
             json_msg = self.ws.recv()
-            msg = json.loads(json_msg)
-            if msg['type'] == "ping":
-                reply = {'type': 'ping'}
-                self.send_msg(reply)
-            elif msg['type'] == "pub":
-                payload = msg['message']
-                if payload['type'] == "setting":
-                    self.handle_settings(payload)
+            print("Websockets msg {}", json_msg)
```
Comment (frankvanbever) [inline ventilator_websocket.py]: Is this print necessary? also you need to call "Websockets msg {}".format(json_msg) for the message to be fromatted into the string in place of the replacement field.

File: ventilator_websocket.py [hunk 2]
```diff
@@ -47,14 +47,18 @@ def run(self, name):
...
+                msg = json.loads(json_msg)
+                if msg['type'] == "ping":
+                    reply = {'type': 'ping'}
+                    self.send_msg(reply)
+                elif msg['type'] == "pub":
+                    payload = msg['message']
+                    if payload['type'] == "setting":
+                        self.handle_settings(payload)
+            except:
+                print("Invalid message from websockets {}", json_msg)
```
Comment (frankvanbever) [inline ventilator_websocket.py]: idem ditto "Invalid message from websockets {}".format(json_msg)

**Review by frankvanbever — CHANGES_REQUESTED** [review]
There's a small issue with the string formatting

File: ventilator_websocket.py (hunk 1 above)
Comment (RenaultR42) [reply ventilator_websocket.py]: You're right, it was for debugging, I forgot to remove it

**Review by frankvanbever — APPROVED** [review]
Looks good to me, merging

**Pull request merged**